source/openapi-admin-v3.yaml

```
@@ -1992,6 +1992,44 @@ paths:
       responses:
         "404":
           description: No IP addresses or CIDR blocks to delete
+  "/groups/{groupId}/apps/{appId}/metrics":
+    get:
+      tags:
+        - metrics
+      operationId: adminGetMetrics
+      summary: Retrieve App Services metrics
+      description: |-
+        Retrieves a variety of metrics, including: Compute Time, Data
+        Transfer, Sync Minutes, Requests Succeeded, Requests Failed.
+      parameters:
+        - "$ref": "#/components/parameters/GroupId"
+        - "$ref": "#/components/parameters/AppId"
+        - name: start
+          in: query
+          description: |-
+            The ISO 8601 date and time of the start of the query period.
+          schema:
+            type: string
+          required: true
+        - name: end
+          in: query
+          description: |-
+            The ISO 8601 date and time of the end of the query period.
+          schema:
+            type: string
+          required: true
+        - name: granularity
+          in: query
+          description: |-
+            Specifies the granularity of the query period as an
+            :wikipedia:`ISO-8601 duration value <ISO_8601#Durations>`. Examples
```

```suggestion
            [ISO-8601 duration value](https://en.wikipedia.org/wiki/ISO_8601#Durations). Examples
```

OpenAPI uses Markdown, not rST